docs/operator-manual/argocd-cm.yaml

```diff
@@ -224,7 +224,11 @@ data:
   # Build options/parameters to use with `kustomize build` (optional)
   kustomize.buildOptions: --load_restrictor none
 
-  # Additional Kustomize versions and corresponding binary paths
+  # Per-version build options and binary paths
+  kustomize.path.v3.9: /custom-tools/kustomize_3_9
```

Comment:
Should we 3 digit version numbers?  (e.g. v3.9.1)

Reply:
Agree, it is good to have real-life example in documentation.